test/internals/logger.test.js

```diff
@@ -74,39 +72,6 @@ test('The logger should reuse request id header for req.id', t => {
   })
 })
 
-test('The logger should add a timestamp if logging to stdout', t => {
-  t.plan(5)
-
-  const stream = split(JSON.parse)
-  const stub = Object.create(loggerUtils)
-  stub.createLogger = function (opts) {
-    return loggerUtils.createLogger(opts, stream)
-  }
-
-  const Fastify = proxyquire('../..', {
-    './lib/logger': stub
-  })
-  const fastify = Fastify({ logger: true })
-
-  stream.once('data', (line) => {
-    t.equal(line.msg, 'incoming request')
-    stream.once('data', (line) => {
-      t.equal(line.msg, 'Not Found')
-      stream.once('data', (line) => {
-        t.equal(line.msg, 'request completed')
-        t.ok(line.responseTime, 'responseTime exists')
-      })
-    })
-  })
-
-  fastify.inject({
-    method: 'GET',
-    url: '/'
-  }, function (err, res) {
-    t.error(err)
-  })
-})
```

Comment:
Why was this removed?

Reply:
We are already checking that in https://github.com/fastify/fastify/blob/834d3ee5817a5e16e6257898fd49aca241c94cf7/test/logger.test.js#L68